tests/Yaapii.Atoms.Tests/Text/EndsWithTests.cs

```diff
@@ -0,0 +1,78 @@
+﻿using Xunit;
+using Yaapii.Atoms.Text;
+
+namespace Yaapii.Atoms.Tests.Text
+{
+    public sealed class EndsWithTests
+    {
+        [Fact]
+        public void FindsTextEnd()
+        {
+            var x =
+                new EndsWith(
+                    new TextOf("Im an text with a realy good end!"),
+                    new TextOf("od end!")
+                );
+            Assert.True(x.Value());
+        }
+
+        [Fact]
+        public void TakesTailAsString()
+        {
+            var x =
+                new EndsWith(
+                    new TextOf("Im an text with a realy good end!"),
+                    "od end!"
+                );
+            Assert.True(x.Value());
+        }
+
+        [Fact]
+        public void FailsOnWrongTail()
+        {
+            var x =
+                new EndsWith(
+                    new TextOf("Im an text with a realy good end!"),
+                    new TextOf("od end")
+                );
+            Assert.False(x.Value());
+        }
+
+        [Fact]
+        public void FailsOnCaseMismatch()
+        {
+            var x =
+                new EndsWith(
+                    new TextOf("Im an text with a realy good end!"),
+                    new TextOf("od enD!")
```

Comment:
Why is this different from the test above? its another byte in the string, why should this fail if the test above works and you do not make a ToLower in your class somewhere? This test is not needed.

Reply:
@Meerownymous 
This test checks if case sensitive is activated or not in the implementing class. In my opinion the test is necessary.

Reply:
No it is not. That would lead to simply test everything that can be done with the text. Test if the end does not match if there is a number. If there is a $ instead of !. If there is a byte and so on. Its overengineering!